src-docs/src/views/guidelines/writing_guidelines.js

```diff
@@ -205,15 +205,27 @@ export const WritingGuidelines = () => {
           </GuideRuleExample>
         </GuideRule>
 
-        <GuideRule heading="Title case for products, solutions, and apps">
+        <GuideRule
+          heading="Title case for proper nouns"
+          description="Proper nouns include product names, solutions, apps, feature tiers, and subscription levels. "
+        >
           <GuideRuleExample
             type="do"
-            text="Product and solution names are always capitalized."
+            text="Product and solution names are always capitalized. Same goes for feature tiers and subscription levels."
             minHeight={200}
           >
-            <EuiTitle size="xs">
-              <span>Elastic APM</span>
-            </EuiTitle>
+            <EuiText size="xs">
+              <span>
+                <ul>
+                  <li>Welcome to Elastic Observability</li>
+                  <li>Elastic APM</li>
+                  <li>You have chosen Security Essentials.</li>
```

Comment:
Very soon, this tier will be renamed to "Security Analytics Essentials" (per https://github.com/elastic/security-team/issues/7925).
```suggestion
                  <li>You have chosen Security Analytics Essentials.</li>
```

Reply:
Thanks Joe! I outdated the comment with another commit but applied it below myself